test/lit/passes/remove-unused-module-elements-refs.wast

```diff
@@ -1,22 +1,14 @@
 ;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
-;; RUN: foreach %s %t wasm-opt --remove-unused-module-elements --closed-world --nominal -all -S -o - | filecheck %s
-;; RUN: foreach %s %t wasm-opt --remove-unused-module-elements                --nominal -all -S -o - | filecheck %s --check-prefix OPEN_WORLD
 
-;; Test both open world (default) and closed world. In a closed world we can do
-;; more with function refs, as we assume nothing calls them on the outside, so
-;; if no calls exist to the right type, the function is not reached.
+;; RUN: foreach %s %t wasm-opt --remove-unused-module-elements --nominal -all -S -o - | filecheck %s
```

Comment:
Was closed-world removed here because the test doesn't pass validation any more? I think it probably still had useful things though. Maybe worth making a new test file for closed world that removes just the non-validating stuff? Otherwise, where are we testing the closed-world-specific code paths in this pass?

Reply:
That's right. Will create a separate test for the closed-world-specific parts.